### PR TITLE
Feature/547 timeline edit plantings dates in sidebar

### PR DESCRIPTION
**Pull request by Bushuo**
Ref: #547  

## Basics

- [X] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [X] The PR is rebased with current master.
- [X] Details of what you changed are in commit messages.
- [X] References to issues, e.g. `close #X`, are in the commit messages.
- [x] The buildserver is happy.

## Checklist

- [x] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [x] I fully described what my PR does in the documentation
      (not in the PR description)
- [x] I fixed all affected documentation
- [x] I fixed all affected decisions
- [ ] I added unit tests for my code
- [x] I added code comments, logging, and assertions as appropriate
- [x] I translated all strings visible to the user
- [x] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [x] I created left-over issues for things that are still to be done
- [x] Code is conforming to [our Architecture](/doc/architecture)
- [x] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [x] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Comment by markus2330** [comment]
jenkins build please

File: doc/CHANGELOG.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - _()_
 - _()_
 - A timeline date can be set and changed by a date picker. Plantings exist relative to a date. _(Paul)_
-- _()_
+- Add/Remove date of plants can be changed in the toolbar. Added `zod` as form validation library. _(Paul)_
```
Comment (markus2330) [inline doc/CHANGELOG.md:35]: Use extra lines for independent parts. Please write a line in solution when to use typeshare and when zod.

File: frontend/src/config/i18n/en/plantings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "add_date": "Added on",
+  "remove_date": "Removed on"
+}
```
Comment (markus2330) [inline frontend/src/config/i18n/en/plantings.json:4]: Shouldn't be "delete planting" translation text in the same file?

**Review by markus2330 — CHANGES_REQUESTED** [review]
Thank you for the PR! We noticed following, please create follow-up issues if it is out-of-scope for this PR:

- [ ] when selecting German, the date format should be German
- [ ] there should be some text explaining what "delete plant" and "remove plant" means
- [ ] "remove plant" and the button "delete plant" should be visually better separated
- [ ] button "delete plant" should be smaller

**Comment by Bushuo** [comment]
@markus2330 
> when selecting German, the date format should be German

The representation of the default browser date input is handled by the browser language. We would need a custom solution if we want to change how it looks currently.

>  there should be some text explaining what "delete plant" and "remove plant" means
 "remove plant" and the button "delete plant" should be visually better separated
 button "delete plant" should be smaller

Everything else I will do later today in this PR :)

**Comment by Bushuo** [comment]
@markus2330 
If we go for a custom solution here I would suggest we use https://ui.shadcn.com/docs which is built upon https://www.radix-ui.com/
It is compatible with tailwindcss (that we use for styling), is headless (can be customized to the very last detail) and can be adopted one component at a time.
I would then also suggest that we swap out the selects and other form components for the ones in shadcn later.
The solution would follow the WAI-ARIA guidelines by default.

**Comment by markus2330** [comment]
I like the idea but it is nothing we should do soon. Let us see if anyone in the user study has English browser settings and selects `De`. :wink: I created  #656

**Comment by Bushuo** [comment]
jenkins build please

**Pull request merged**